schema/iiif_3_0.json

```diff
@@ -308,20 +308,126 @@
                             "logo": {
                                 "type": "array",
                                 "items": { "$ref": "#/classes/resource" }
-                            },    
+                            },
                             "seeAlso": { "$ref": "#/classes/seeAlso" }
                         }
                     }
                 ]
             }
         },
+        "collection": {
+            "allOf": [
+                { "$ref": "#/types/class" },
+                {
+                    "type": "object",
+                    "properties": {
+                        "type": {
+                            "type": "string",
+                            "pattern": "^Collection"
+                        },
+                        "metadata": { "$ref": "#/classes/metadata" },
+                        "summary": { "$ref": "#/types/lngString" },
+                        "requiredStatement": { "$ref": "#/types/keyValueString" },
+                        "rights": {
+                            "type": "string",
+                            "format": "uri",
+                            "pattern": "^http.*$"
+                        },
+                        "navDate": {
+                            "type": "string",
+                            "format": "date-time"
+                        },
+                        "provider": { "$ref": "#/classes/provider" },
+                        "seeAlso": { "$ref": "#/classes/seeAlso" },
+                        "thumbnail": {
+                            "type": "array",
+                            "items": { "$ref": "#/classes/resource" }
+                        },
+                        "homepage": { "$ref": "#/classes/homepage" },
+                        "behavior": { "$ref": "#/classes/behavior" },
+                        "partOf": { "$ref": "#/classes/partOf" },
+                        "items": {
+                            "type": "array",
+                            "items": {
+                              "oneOf": [
+                                { "$ref": "#/classes/manifest" },
+                                { "$ref": "#/classes/collection" }
+                              ]
+                            }
+                        },
+                        "annotations": {
+                            "type": "array",
+                            "items": {
+                                "$ref": "#/classes/annotationPage"
+                            }
+                        }
+                    }
+                }
+            ]
```

Comment:
We should require label as this is optional in #/types/classes and is required in: https://iiif.io/api/presentation/3.0/#51-collection